frontend/src/app/components/wp-single-view-tabs/tab/tab.component.ts

```diff
@@ -0,0 +1,6 @@
+import { Component } from '@angular/core';
+import { WorkPackageResource } from 'core-app/modules/hal/resources/work-package-resource';
+
+export interface TabComponent extends Component {
```

Comment:
The standard approach is to create components by using the @Component decorator.